extensions/amp-story/1.0/amp-story-interactive-binary-poll.css

```diff
@@ -78,16 +76,7 @@
   left: 0px !important;
   right: 0px !important;
   height: 100% !important;
-  animation: flashBG forwards var(--animation-time) var(--ease-out-curve);
-}
-
-@keyframes flashBG {
-  0% {
-    opacity: .4;
-  }
-  100% {
-    opacity: .2;
-  }
+  animation: flashBG forwards var(--i-amphtml-interactive-animation-time) var(--i-amphtml-interactive-ease-out-curve);
 }
```

Comment:
Moved into amp-story.css

Reply:
We should change `flashBG` to something like `--i-amphtml-interactive-flash-background` to match the @keyframes naming conventions used in the codebase.

(I know I originally named this variable :) but I'm just catching this now)